conda_forge_tick/utils.py

```diff
@@ -536,6 +536,38 @@ def _parse_recipes(
     RecipeTypedDict
         A dict conforming to conda-build's rendered output
     """
+    # some heuristics to find the "top-level" name and version
+    # we use these in the following order
+    # 1. Does the output name match the "name" field in the context block
+    #    if present?
+    # 2. Does the output version match the "version" field in the
+    #    context block if present?
+    # 3. Does the output name match the feedstock-name in the extra
+    #    field if present?
+    if len(validated_recipes) > 0:
```

Comment:
Should this be only when there's more than one?

```suggestion
    if len(validated_recipes) > 1:
```